### PR TITLE
Es6 modules

### PR DESCRIPTION
**Pull request by K-Karneboge**
image

### CODE REVIEW

File: es6-modules/src/components/todo-app.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      createElement('h1', { class: 'mb-4' }, 'JS Todo'),
+      this.todoForm.render(),
+      this.todoList.render(this.todos)
+    );
+    const { nextId, todos } = this;
+    this.onUpdate({ nextId, todos });
+  }
+}
+
+export { TodoApp as default };
```
Comment (scbowler) [inline es6-modules/src/components/todo-app.js]: Better to write this this way:
```JAVASCRIPT
export default TodoApp;
```

File: es6-modules/src/components/todo-form.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        ])
+      ])
+    );
+    this.element.addEventListener('change', this.handleChange);
+    this.element.addEventListener('submit', this.handleSubmit);
+    return this.element;
+  }
+}
+
+export { TodoForm as default };
```
Comment (scbowler) [inline es6-modules/src/components/todo-form.js]: Same here:
```JAVASCRIPT
export default TodoForm;
```

File: es6-modules/src/components/todo-list.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      createElement('ul', { class: 'list-group shadow-sm mb-4' }, todos.map(todo => {
+        return this.renderTodoItem(todo);
+      }))
+    );
+    todoList.addEventListener('change', this.handleChange);
+    return todoList;
+  }
+}
+
+export { TodoList as default };
```
Comment (scbowler) [inline es6-modules/src/components/todo-list.js]: Same here
```JAVASCRIPT
export default TodoList;
```

File: es6-modules/src/lib/create-element.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  toArray(children).forEach(child => {
+    if (!(child instanceof HTMLElement)) {
+      child = document.createTextNode(child);
+    }
+    parent.appendChild(child);
+  });
+  return parent;
+}
+
+export { createElement as default };
```
Comment (scbowler) [inline es6-modules/src/lib/create-element.js]:
```JAVASCRIPT
export default createElement;
```

File: es6-modules/src/lib/noop.js [hunk 5]
```diff
@@ -0,0 +1,3 @@
+function noop() {}
+
+export { noop as default };
```
Comment (scbowler) [inline es6-modules/src/lib/noop.js]:
```JAVASCRIPT
export default noop;
```

File: es6-modules/src/lib/to-array.js [hunk 6]
```diff
@@ -0,0 +1,5 @@
+function toArray(value) {
+  return Array.isArray(value) ? value : [value];
+}
+
+export { toArray as default };
```
Comment (scbowler) [inline es6-modules/src/lib/to-array.js]:
```JAVASCRIPT
export default toArray;
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, just need to update your `export default`s. Let me know if you have any questions.

**Comment by K-Karneboge** [comment]
image

made adjustments.

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**